src/model_signing/_cli.py

```diff
@@ -355,12 +366,15 @@ def _verify_sigstore(
         identity=identity, oidc_issuer=identity_provider
     )
     signature_contents = sigstore.SigstoreSignature.read(signature)
-    _serialize_and_verify(model_path, verifier, signature_contents, signature)
+    _serialize_and_verify(
+        model_path, verifier, list(ignore_paths), signature_contents, signature
```

Comment:
If you make it a `list`/`Sequence` in the type then `list` would not be needed here.

Reply:
The `type=list[...]` is unfortunately ignored and I always get `<class 'tuple'>`.